storey/writers.py

```diff
@@ -10,29 +10,38 @@
 from .flow import Flow, _termination_obj, _split_path
 
 
-class WriteCSV(Flow):
+class WriteToCSV(Flow):
     """
     Writes events to a CSV file.
 
     :param path: path where CSV file will be written.
     :type path: string
-    :param event_to_line: function to transform an event to a CSV line (represented as a list).
-    :type event_to_line: Function (Event=>list of string)
-    :param header: a header for the output file.
-    :type header: list of string
-    :param name: Name of this step, as it should appear in logs. Defaults to class name (WriteCSV).
+    :param columns: fields to be written to CSV. Will be written as the file header if write_header is True. Will be extracted from
+    events when an event is a dictionary (lists will be written as is). Optional. Defaults to None (will be inferred if event is dictionary,
+    metadata_columns must be False).
```

Comment:
why ` metadata_columns must be False`?

Reply:
Because metadata columns tell you which columns should get which metadata. If there are no known columns, you can't select which to fill with metadata.

Reply:
Discussed and decided to allow this use case and overwrite in case of naming conflict.